### PR TITLE
Try to compile benches in every crate of the workspace

### PR DESCRIPTION
**Pull request by kpp**

### CODE REVIEW

File: tokio-threadpool/benches/basic.rs [hunk 1]
```diff
@@ -34,22 +41,26 @@ mod us {
...
+                });
+                let pool_sender = pool.sender().clone();
+                spawn(fut, &pool_sender).wait().unwrap();
             }
 
             let _ = rx.recv().unwrap();
         });
     }
 
+    /* TODO: fix bench
```
Comment (kpp) [inline tokio-threadpool/benches/basic.rs]: I don't know what this bench should do so I skipped it.

File: tokio-threadpool/benches/basic.rs [hunk 2]
```diff
@@ -80,19 +91,15 @@ mod us {
...
 
-// In this case, CPU pool completes the benchmark faster, but this is due to how
-// CpuPool currently behaves, starving other futures. This completes the
-// benchmark quickly but results in poor runtime characteristics for a thread
-// pool.
-//
-// See alexcrichton/futures-rs#617
-//
-mod cpupool {
+/* TODO do we need this bench?
```
Comment (kpp) [inline tokio-threadpool/benches/basic.rs]: I don't know what this bench should do so I skipped it.

File: tokio-threadpool/benches/depth.rs [hunk 3]
```diff
@@ -33,30 +32,31 @@ mod us {
...
 
-            spawn(sched_tx.clone(), res_tx, super::ITER);
+            spawn(tx.clone(), res_tx.clone(), super::ITER);
             res_rx.recv().unwrap();
         });
     }
 }
 
-mod cpupool {
+/* TODO do we need this bench?
```
Comment (kpp) [inline tokio-threadpool/benches/depth.rs]: I don't know what this bench should do so I skipped it.

**Comment by carllerche** [comment]
Thanks! I tweaked the commit and opened a new PR: #188.

**Pull request closed without merging**